build/testing/integration/api/api.go

```diff
@@ -230,6 +231,10 @@ func API(t *testing.T, ctx context.Context, client sdk.SDK, namespace string, au
 
 		t.Log("Get flag \"test\" and check variants.")
 
+		// TODO: don't love this..
+		// wait for flag cache to expire when running with cache
+		time.Sleep(750 * time.Millisecond)
```

Comment:
Yeah... I was wondering what the alternative was but the alternatives are dont cache haha.